### PR TITLE
✨ NEW: Option for download button

### PR DESCRIPTION
**Pull request by bknaepen**
This relates to the following open issue:

https://github.com/executablebooks/sphinx-book-theme/issues/244

I am completely new to this package but this proposal seems to be functional. I just duplicated lines I found for the issue button. 

The tests are failing but I was unable to fix this.

### CODE REVIEW

File: src/jinja/theme.conf.j2 [hunk 1]
```diff
@@ -17,6 +17,7 @@ navbar_footer_text =
 extra_navbar = Theme by the <a href="https://ebp.jupyterbook.org">Executable Book Project</a>
 extra_footer =
 use_issues_button = False
+use_download_button = False
```
Comment (choldgraf) [inline src/jinja/theme.conf.j2:20]: let's set this to `True` by default

Comment (bknaepen) [reply src/jinja/theme.conf.j2:20]: That did it :-)

File: sphinx_book_theme/theme.conf [hunk 2]
```diff
@@ -17,6 +17,7 @@ navbar_footer_text =
 extra_navbar = Theme by the <a href="https://ebp.jupyterbook.org">Executable Book Project</a>
 extra_footer =
 use_issues_button = False
-use_repository_button = False
+use_download_button = False
```
Comment (choldgraf) [inline sphinx_book_theme/theme.conf:20]: hmm - `use_repository_button` should remain `False` (the default that we currently have), and `use_download_button` should be `True` (also our current default behavior)

Can you also add documentation for this feature to https://github.com/executablebooks/sphinx-book-theme/blob/master/docs/configure.md ?

Comment (bknaepen) [reply sphinx_book_theme/theme.conf]: Sorry, I obviously edited the wrong line 🤦🏻‍♂️. I hope I got it right this time...

I'll proceed to the documentation and be back.

**Comment by pradyunsg** [comment]
Another `The message was: Cell execution timed out.`. Closing-reopening to re-trigger CI.

**Comment by choldgraf** [comment]
Thanks very much for this! RTD timeout errors fixed so let's get it in!

**Comment by bknaepen** [comment]
@choldgraf I'm glad this feature was found useful, thanks for helping out :-). You mentioned above that a validation test would also be useful. I could work on this in the coming days when time permits but I need to dig a bit more on how to do this. What do you think?

**Comment by choldgraf** [comment]
that would be great @bknaepen . I'd recommend checking out how the tests work for similar features: https://github.com/executablebooks/sphinx-book-theme/blob/master/tests/test_build.py#L217 and try to recreate those patterns. Wanna give that a shot?

**Comment by bknaepen** [comment]
@choldgraf Yes I will try ;-)

**Pull request merged**